### PR TITLE
changelog for 6.24.0 [skip ci]

### PR DESCRIPTION
**Pull request by hzoo**
cc @rwjblue, @izaakschroeder

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,65 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
...
 
+## 6.24.0 (2017-03-13)
+
+A quick release for 2 features:
+
+- Thanks to @rwjblue, there is a `noInterop` option for our `es2015-modules` transforms to remove the `interopRequireDefault` helper.
+
+This also helps [ember-cli migrate to Babel 6](https://github.com/ember-cli/ember-cli/pull/6828).
+
+- @izaakschroeder has adding `dirname` to the preset constructor which presets can use to resolve things relative to files.
```
Comment (Kovensky) [inline CHANGELOG.md]: added

**Comment by izaakschroeder** [comment]
🚀

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,65 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See [CHANGELOG - 6to5](CHANGELOG-6to5.md) for the pre-4.0.0 version changelog.
 
+## 6.24.0 (2017-03-13)
+
+A quick release for 2 features:
+
+- Thanks to @rwjblue, there is a `noInterop` option for our `es2015-modules` transforms to remove the `interopRequireDefault` helper.
```
Comment (existentialism) [inline CHANGELOG.md]: > there is _now_ a `noInterop` option

> for our `es2015-modules` _transform_ that removes the ``interopRequireDefault` helper

Comment (rwjblue) [inline CHANGELOG.md]: Not a big deal, but it also removes `interopRequireWildcard` too.

**Comment by hzoo** [comment]
Since probably no one is going to understand those changes (I had to read the prs again 🙂), added examples

**Comment by hzoo** [comment]
Dang didn't see notifications for approvals in the slack 🙂. Oh that would be an interesting bot idea

**Pull request merged**